schemas/gtfs_realtime_source_schema.json

```
@@ -35,6 +35,20 @@
       },
       "description": "A list of the static sources that the real time source is associated with, represented by their MDB source IDs. Although optional, if not provided, the location of this source will be set to unknown."
     },
+    "features": {
+      "type": "array",
+      "description": "A list of features that the source supports.",
+      "items": {
+        "type": "string",
+        "enum": ["occupancy"]
+      }
+    },
+    "status": {
+      "type": "string",
+      "description": "The status of the source which can be used to determine if/how to include it in public trip-planning. If not provided, source is assumed to be active. Should be one of: active: The source is active and should be included in public trip-planning. inactive: The source is inactive / hasn't been updated recently and should be included in public trip-planning at risk of providing outdated information. deprecated: The source is explicitly deprecated and should not be included in public trip-planning. development: The source is in development or being used to test a specific feature and should not be included in public trip-planning.",
+      "enum": ["active","deprecated","inactive","development"]
+    }
+  },
```

The JSON file is invalid.
```suggestion
    },
```